README.md

```diff
@@ -0,0 +1,86 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic "Hello world" page that illustrate the use of tools and best practices for future Microverse projects.
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- Stylelint, Github Actions
+
+## Live Demo
+
+[Live Demo Link - coming soon]()
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Node and NPM
+- Web browser (Google chrome)
+- Code Editor (VSCode)
+
+### Development (Running locally)
+
+- Clone the project
+
+```bash
+git clone https://github.com/rotimiazeez/hello-world.git
```

Comment:
Can you please change the git URL to your git repo?
Try as much as possible to check your code before committing to avoid this happening.